README.md

```diff
@@ -152,3 +152,289 @@ commit it in this repo.
+the unfinished [PR #49](https://github.com/input-output-hk/cardano-parts/pull/49).
+
+We'll focus on updating based on the latest merged PR (#48).
+
+### Step 4: Update `cardano-parts` Version
+
+To update `cardano-parts` to the latest version, run:
+
+```bash
+nix flake update cardano-parts
+```
+
+This updates to the latest version from the `main` branch. If you want to
+point to a specific branch or revision, manually update the `flake.nix` file.
+After updating, you can use the following commands to verify changes:
+
+- Run `menu` to confirm that the `cardano-node` version has changed.
+- Run `just` to list useful commands.
+
+### Step 5: Apply Changes from the PR
+
+Review the PR details for any changes or recommendations. For example, PR
+[#48](https://github.com/input-output-hk/cardano-parts/pull/48) modifies the
+following files:
+
+```
+Justfile                                                                # Adds IPv6 recipe support
+flake/colmena.nix                                                       # Adds staticIpv6 declaration
+flake/opentofu/cluster.nix                                              # Adds IPv6 Terraform support
+flake/opentofu/grafana/alerts/cardano-node-divergence.nix-import        # Updates alerts for new tracing metrics
+flake/opentofu/grafana/alerts/cardano-node-forge.nix-import             # Updates alerts for new tracing metrics
+flake/opentofu/grafana/alerts/cardano-node-quality.nix-import           # Updates alerts for new tracing metrics
+flake/opentofu/grafana/alerts/cardano-node.nix-import                   # Updates alerts for new tracing metrics
+flake/opentofu/grafana/dashboards/cardano-node-new-tracing.json         # Updates dashboard for new tracing metrics
+flake/opentofu/grafana/dashboards/cardano-node-p2p-new-tracing.json     # Updates dashboard for new tracing metrics
+flake/opentofu/grafana/dashboards/cardano-performance-new-tracing.json  # Updates dashboard for new tracing metrics
+```
+
+Focus on inspecting changes to `flake/colmena.nix` and `flake/cluster.nix`, as
+these are important. Use the following commands to review and apply changes:
+
+- To diff a file, use:
+
+  ```bash
+  just template-diff "$FILE"
+  ```
+
+- To apply changes if the diff looks correct, use:
+
+  ```bash
+  just template-patch "$FILE"
+  ```
+
+For other files, you can clone them directly by running:
+
+```bash
+just template-clone "$FILE"
+```
+
+**Note**: Some files listed in the PR may not exist in your deployment. These
+can typically be ignored unless explicitly mentioned in the PR description.
+
+### Step 6: Apply the Changes
+
+After updating the necessary files, apply the changes with:
+
+```bash
+just apply
+```
+
+For specific machines, use something similar to:
+
+```bash
+just apply "'mainnet*'"
+```
+
+_Important_: If changes were made to `opentofu` or `grafana` files, you also
+need to run:
+
+```bash
+just tf grafana apply
```

Comment:
`just tf grafana apply` for grafana related resource changes (ex: dashboards, alerts, etc) and
`just tf cluster apply` (or `just tf apply` which defaults to `cluster` workspace) for cluster resources changes, such as aws ec2, route53, etc.